libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -326,6 +327,26 @@ def _identifying_params(self) -> Dict[str, Any]:
             "default_request_timeout": self.default_request_timeout,
         }
 
+    def _get_ls_params(
+        self, stop: Optional[List[str]] = None, **kwargs: Any
+    ) -> LangSmithParams:
+        """Get the parameters used to invoke the model."""
+        params = super()._get_invocation_params(stop=stop, **kwargs)
+        ls_temperature = params.pop("temperature", self.temperature)
+        ls_max_tokens = params.pop("max_tokens", self.max_tokens)
+        ls_stop = stop or params.pop("stop", None)
+        _ = params.pop("model", None)
+        ls_params = LangSmithParams(
+            ls_provider="anthropic",
+            ls_model_name=self.model,
+            ls_model_type="chat",
+            ls_temperature=ls_temperature,
+            ls_max_tokens=ls_max_tokens,
+            ls_stop=ls_stop,
+        )
+        ls_params.update(cast(LangSmithParams, params))
```

Comment:
why doing this?

Reply:
I `.update(params)` so we don't lose model kwargs that we haven't standardized yet.